recipe/conda_build_config.yaml

```diff
@@ -1,2 +0,0 @@
-numpy:      # [aarch64]
```

Comment:
@qwhelan do you know why this file was needed? It seems to be causing issues with re-rendering. Is it ok if we get rid of it?

Reply:
@qwhelan may I ask you to take a look, please?
We can't support arm jupyter docker-stacks easily without aarch64 python 3.9 support of this package.